scripts/convert.js

```diff
@@ -1,37 +1,35 @@
 // to use this run `node ./index.js`
 
-// const extract = require('string-extract-class-names')
-const extract = require('./custom-string-extract-class-names'); // original filters too short classes (e.g .b, .i)
+const extract = require('./custom-string-extract-class-names') // original filters too short classes (e.g .b, .i)
+const postcss = require('postcss')
+const _ = require('lodash')
+const fs = require('fs')
+const CleanCSS = require('clean-css')
 
-const postcss = require('postcss');
-const _ = require('lodash');
-const exportPath = './src/Tachyons/Classes.elm';
+const elmHelpersPath = './src/Tachyons.elm'
+const elmClassesPath = './src/Tachyons/Classes.elm'
 
-fs = require('fs');
-
-const css = fs.readFileSync('./scripts/tachyons.css', 'utf8'); // minificated version introduces problems with compound selectors
-
-const root = postcss.parse(css);
-
-const classObjs = {};
-
-const defaultIndentation = ' '.repeat(4);
+const css = fs.readFileSync('./scripts/tachyons.css', 'utf8')
+const minifiedCSS = new CleanCSS().minify(css);
+const root = postcss.parse(css)
+const classObjs = {}
+const defaultIndentation = ' '.repeat(4)
 
 const ruleFormatter = rule => {
-  let def = rule.toString().replace('{-', '{ -');
-  def = setCorrectIndentation(def);
-  return def;
-};
```

Comment:
Sorry, I don't list this explicitly anywhere by to keep things consistent just run [Prettier](https://github.com/prettier/prettier) with the default options

Reply:
Nice!
Ok I prettified the scripts and added a note on the README.